core/src/main/web/app/helpers/helper.js

```diff
@@ -199,6 +199,13 @@
           console.error("Current app doesn't support updateStatus");
         }
       },
+      getStatus: function() {
+        if (getCurrentApp().getStatus) {
```

Comment:
It would certainly be more specific, but in my (obviously humble) opinion it is unnecessary.  It would be useful if there were some case where `getStatus` was an existing attribute but wasn't a function (which would be a terrible thing).  If that isn't the case, isn't it easier to just test the falsiness of the attribute?